docs/pages/10_view.adoc

```diff
@@ -0,0 +1,629 @@
+
+*Syntax:*
+
+[subs="quotes"]
+----
+[purple]#**link**# &lt;view> &lt;name> &#x5B;[purple]##**<=**## &lt;expression>]
+    &#x5B;[purple]##**required:**##([purple]##**true**##|[purple]##**false**##)]
+    &#x5B;[purple]##**enabled:**##&lt;enabled>]
+    &#x5B;[purple]##**default:**##&lt;default>]
+    &#x5B;[purple]##**choices:**##&lt;choices>]
+    &#x5B;[purple]##**selector:**##&lt;row>]
+    &#x5B;[purple]##**eager:**##([purple]##**true**##|[purple]##**false**##)]
+    &#x5B;[purple]##**create:**##([purple]##**true**##|[purple]##**false**##)]
+    &#x5B;[purple]##**delete:**##([purple]##**true**##|[purple]##**false**##)]
+    &#x5B;[purple]##**update:**##([purple]##**true**##|[purple]##**false**##)][purple]#**;**#
+
+    &#x5B;[purple]##**label:**##&lt;label>]
+    &#x5B;[purple]##**icon:**##&lt;icon>]
+    &#x5B;[purple]##**hidden:**##&lt;hidden>]
+    &#x5B;[purple]##**width:**##&lt;width>]
+    &#x5B;[purple]##**text:**##&lt;text>][purple]#**;**#
+----
+
+The syntax for defining a view link includes various modifiers that control its behavior and appearance. These modifiers allow you to specify whether the link is required, enabled, or has a default value. You can also define the choices available for selection, the label and icon displayed for the link, and its width.
+
+Modifiers such as `label`, `icon`, `hidden`, and `width` can be used to customize the common appearance and behavior of the component. See more about them in <<Common visual modifiers>> chapter.
+
+The `text` modifier sets the text to be displayed for the link. This is an alternative to the `label` modifier and is used specifically for specifying the text displayed on the link.
+
+Since the view link is also a single transfer relation, its basic syntax is similar to that of a transfer relation. However, unlike a transfer relation, the cardinality of a view link cannot be specified; it always targets a single view. The view link focuses on enhancing the user experience by providing intuitive navigation and visual cues.  To get specific details about transfer relation, please refer to the section on the xref:07_transfer.adoc[Transfer model].
+
+*Example:*
+
+[source,jsl]
+----
+link ProductListView productList text:"Back to shopping";
+----
+
+In this example, `ProductListView` represents the target view that the link navigates to. `productList` is the name of the link, and `text:"Back to shopping"` specifies the text displayed for the link.
+
+=== View table
+
+A view table is a graphical representation within a user interface that displays a collection of rows in a tabular format. It organizes data into rows and columns, making it easier for users to view and interact with structured information. The full list of modifiers can be found in the xref:appx_syntax.adoc#view-table[Appendix].
```

Comment:
I can't find the specification for selector tables on tables, e.g. the add selector.

Reply:
Do table create operations create Row instances or default link target instances? If the later, then who's responsible of the creation of the intermediate relation?